CONTRIBUTING.md

```diff
@@ -2,32 +2,45 @@
 
 ## Style guidelines
 
-We are currently aspiring to follow the [Agda standard library style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md) as much as reasonable. Since some of our code will be rendered into a PDF, the formatting of the PDF takes priority over formatting of the code, so deviations are to be expected.
+We are currently aspiring to follow the [Agda standard library style guide][] as much as reasonable. Since some of our code will be rendered into a PDF, the formatting of the PDF takes priority over formatting of the code, so deviations are to be expected.
 
 ## Setup with emacs
 
-We rely on standard Agda version 2.6.3 and a patched version of the Agda Standard Library; this makes setup more difficult. You can install the correct version of Agda and the stdlib using `nix-env -iA agda -f .`, but this is a global install which you may not want if you also have other Agda projects.
+We use Agda version 2.6.3 and a patched version of the Agda Standard Library; this makes setup more difficult. You can install the correct version of Agda and the stdlib using `nix-env -iA agda -f .`, but this is a global install which you may not want if you also have other Agda projects.
 
 To install Agda locally and use that install with emacs, you can do the following:
 
-- Put the version of Agda specified in the nix file somewhere: `nix-build -A agda -o ~/IOHK/ledger-agda`.
-- Put the following into your init file (highlight and `M-x eval-region` to load it without restarting emacs). Note that this assumes that your regular install of Agda is in your path with the name `agda` and version `2.6.2.2`, otherwise you'll have to edit these variables.
-```
-(setq my/ledger-agda-name "~/IOHK/ledger-agda")
-(defun my/toggle-ledger-agda ()
-  (interactive)
-  (if (string-equal agda2-program-name "agda")
-      (setq agda2-version      "2.6.3"
-            agda2-program-name (concat my/ledger-agda-name "/bin/agda"))
-    (setq agda2-version      "2.6.2.2"
-          agda2-program-name "agda"))
-  (agda2-restart))
-(with-eval-after-load 'agda2-mode (define-key agda2-mode-map (kbd "C-c C-x C-t") 'my/toggle-ledger-agda))
-```
+-  Build `agda` and `agda-mode` binaries and put symlinks to them somewhere in your path (e.g., ~/IOHK/ledger-agda`):
+
+   ```
+   mkdir -p ~/IOHK
```

Comment:
@WhatisRT Don't we need to make sure the `~/IOHK` directory exists (or else create it) in order for the `nix-build -A agda -o ~/IOHK/ledger-agda` command to succeed?

Reply:
Also, I think `export PATH = ~/IOHK/ledger-agda/bin:$PATH`, or something like it, is necessary, yes?

Reply:
That `export` shouldn't be necessary, especially since we point emacs to the path manually. It also doesn't do anything outside the shell it's currently run from.

For making sure the directory exists, if we're that explicit about it maybe we should make it clear that it's a 'put it wherever you like' situation. Something like `my/favourite/directory/ledger-agda`

Reply:
In particular, we don't want them in the PATH, since then they will collide with a global install